hb_report/utillib.py

```diff
@@ -1439,6 +1440,25 @@ def say_ssh_user():
         return constants.SSH_USER
 
 
+def sbd_info():
+    """
+    sbd information
+    """
+    out_string = "#####SBD info:\n\n"
+    sbd_device = grep("(^|^\s*)SBD_DEVICE=.*", infile=constants.SBDCONF)
```

Comment:
the sbd device can support multiple devices up to 3, it is like: 
SBD_DEVICE="/dev/sda1;/dev/sdb1;/dev/sdc1"

this need to be considered as well